backend/create_superuser.py

```
@@ -0,0 +1,82 @@
+        print("   Password: [hidden]")
+        print("   Admin URL: http://127.0.0.1:8000/admin/")
+
+        # Optional: write the generated password to a local file (ignored by git)
+        if os.environ.get('WRITE_ADMIN_PASSWORD_TO_FILE', 'false').lower() == 'true':
+            with open('.admin_credentials', 'w') as f:
+                f.write(f"username={username}\nemail={email}\npassword={password}\n")
+            print("INFO: Credentials written to .admin_credentials")
+
+        return user
+
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)